lib/teamsnap.rb

```diff
@@ -34,7 +34,7 @@ def init(opts = {})
       self.root_client = TeamSnap::Client.new(:token => token)
 
       ##   Make the apiv3 root call. collection is parsed JSON
-      collection = TeamSnap.run(root_client, :get, "/", {})
+      collection = TeamSnap.run(root_client, :get, self.url, {})
```

Comment:
If you try to hit `https://api.teamsnap.com` it gives you an error.
You actually have to hit `https://api.teamsnap.com/v3/` to get the root.